calendar_v17/odoo_office365_calendar/models/calendar_event.py

```
@@ -0,0 +1,651 @@
+            attandees = self.env["res.partner"].browse(partner.id)
+            attandees_list.append(
+                {
+                    "Type": "Required",
+                    "EmailAddress": {
+                        "Name": attandees.name,
+                        "Address": attandees.email,
+                    },
+                }
+            )
+        for alarm in self.alarm_ids:
+            nofication = self.env["calendar.alarm"].search(
```

Ruff (0.11.9)

Local variable `nofication` is assigned to but never used

Remove assignment to unused variable `nofication`

(F841)